src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Customizations/Skillsets/SkillConstructors/EntityRecognitionSkill_BackCompatCtror.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Search.Models
+{
+    using System.Collections.Generic;
+
+    public partial class EntityRecognitionSkill
```

Comment:
Please include a comment for future maintainers explaining why this is here, and when it will be safe to remove it.

Reply:
Same for all the other files